### PR TITLE
 CBG-222: Decreased default revsLimit

### PR DESCRIPTION
**Pull request by JRascagneres**

### CODE REVIEW

File: db/database_test.go [hunk 1]
```diff
@@ -860,6 +860,40 @@ func TestConflicts(t *testing.T) {
 		branched: true})
 }
 
+func TestConflictRevLimit(t *testing.T) {
+
+	//Test Default Is the higher of the two
+	db, _ := setupTestDB(t)
+	assert.Equal(t, uint32(DefaultRevsLimitConflicts), db.RevsLimit)
+
```
Comment (adamcfraser) [inline db/database_test.go:868]: Add teardown for db.

**Review by adamcfraser — CHANGES_REQUESTED** [review]
As discussed, modify the warning being emitted in server_context.go when the revslimit is < 100.

File: rest/server_context.go [hunk 2]
```diff
@@ -664,7 +664,7 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 			}
 
 			if dbcontext.RevsLimit < 100 {
-				base.Warnf(base.KeyAll, "Setting the revs_limit (%v) to less than 100 may have unwanted results when documents are frequently updated. Please see documentation for details.", dbcontext.RevsLimit)
+				base.Warnf(base.KeyAll, "Setting the revs_limit (%v) to less than 100, whilst having allow conflicts set to true, may have unwanted results when documents are frequently updated. Please see documentation for details.", dbcontext.RevsLimit)
```
Comment (JFlath) [inline rest/server_context.go]: Would be nice to change this to `allow_conflicts`, to make it clear what needs to be changed.

File: rest/server_context.go [hunk 3]
```diff
@@ -664,7 +664,7 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 			}
 
 			if dbcontext.RevsLimit < 100 {
```
Comment (JFlath) [inline rest/server_context.go]: (Sorry, should have got this in in one go) - should we extract this as a constant, so it's not a magic number buried in the code? Could also then sub it into the warning message...

Comment (JRascagneres) [reply rest/server_context.go]: Swapped this out to use the default revs limit allow conflict constant here

**Review by adamcfraser — APPROVED** [review]

**Pull request merged**